tests/client/factory_test.py

```diff
@@ -322,11 +322,7 @@ def test_check_hostname_false_when_host_is_ip(self):
     @mock.patch("esrally.client.asynchronous.RallyAsyncElasticsearch")
     def test_create_async_client_with_api_key_auth_override(self, es):
         hosts = [{"host": "localhost", "port": 9200}]
-        client_options = {
-            "use_ssl": True,
-            "verify_certs": True,
-            "http_auth": ("user", "password"),
-        }
+        client_options = {"use_ssl": True, "verify_certs": True, "http_auth": ("user", "password"), "max_connections": 600}
```

Comment:
Adding a trailing comma after `max_connections` would have produced a better diff.

Reply:
Thanks. Addressed in https://github.com/elastic/rally/pull/1669/commits/80623af64fce687b94513f1c309d73cdafa1db2b.